CMakeLists.txt

```diff
@@ -7,6 +7,16 @@ if(POLICY CMP0091)
   cmake_policy(SET CMP0091 NEW)
 endif()
 
+#
+# We use use sometimes ;-connected AND syntax in cmake_dependent_option,
```

Comment:
Actually, the `;` syntax is still supported in 3.22+ as well:
https://cmake.org/cmake/help/v3.22/module/CMakeDependentOption.html#command:cmake_dependent_option